checked/bytes.go

```diff
@@ -54,6 +54,7 @@ func NewBytes(value []byte, opts BytesOptions) Bytes {
 		value: value,
 	}
 	b.SetFinalizer(b)
+	b.TrackObject(b.value)
```

Comment:
That wouldn't work necessarily though as there could easily be concurrent goroutines that it would have side affects on (i.e. you wouldn't just be turning it on for the code between those calls, other code could receive the side affects too).